Setup.tex

```diff
@@ -44,7 +44,7 @@ \section{General Setup}
 
 \section{Welcome Reception}
 \label{sec:welcome_reception}
-Traditionally --since Eindhoven 2013-- the RoboCup@Home holds an own \iterm{welcome reception} in addition to the official opening ceremony. During the welcome reception, a \iterm{poster session} is held in which teams present their research foci and latest results (see~\refsec{sec:poster_teaser_session}).
+Traditionally --since Eindhoven 2013-- the RoboCup@Home holds their own \iterm{welcome reception} in addition to the official opening ceremony. During the welcome reception, a \iterm{poster session} is held in which teams present their research foci and latest results (see~\refsec{sec:poster_teaser_session}).
```

Comment:
*its* would be most correct I think, but their make sense since there are several leagues. I don't know.

Reply:
`its` Or `their` are both better than the original “an own”.

We have several sub leagues but only one OC/TC/EC so it is difficult to say what is correct here.

The RC@Home leagues hold their own.
The RC@Home league holds its own.

Reply:
I'm fine with either. Accepting change.